libfwupdplugin/fu-plugin.h

```diff
@@ -61,6 +72,291 @@ typedef enum {
 	FU_PLUGIN_VERIFY_FLAG_LAST
 } FuPluginVerifyFlags;
 
+/* one day this will be part of FuPluginClass */
```

Comment:
If you're breaking ABI already anyway with this change and need to bump up the soname, do this now.

Reply:
Done? or not?

Reply:
Ohh, sorry, replied to the wrong one. I didn't do this because we can't -- the FuPluginClass is per-type, and not per-instance. I don't think it makes sense to have a FuDfuPlugin GType with vfuncs, at least not at this point, as we'd then need a dlsym entrypoint like now to return the custom GType. It's a whole ton of boilerplate without much benefit IMHO. Maybe we just remote the comment?

Reply:
Yeah remove comment then

Reply:
Done